HackYourFuture-CPH/chattie
# Feature/#134/frontend style mock9

![task134_desctop](https://user-images.githubusercontent.com/50618208/94562627-d85f2a80-0265-11eb-9d26-4795ecf76cb7.png)
![task134_mobile](https://user-images.githubusercontent.com/50618208/94562629-d8f7c100-0265-11eb-91a9-933852f71926.png)

# Description

I added buttons and a Logo, styling all.  I left the menu on the left side. Will fix it 

Fixes # 134

# How to test?

For now, you can take a look on screenshots. This story very connected to story #174 ( fixe style in index.css) (only when this task is merged you can check a result of this story) 
I tested it by npm run dev

# Checklist
